seq2seq/evaluator/predictor.py

```diff
@@ -42,3 +43,31 @@ def predict(self, src_seq):
         tgt_id_seq = [other['sequence'][di][0].data[0] for di in range(length)]
         tgt_seq = [self.tgt_vocab.itos[tok] for tok in tgt_id_seq]
         return tgt_seq
+
+    def predict_n(self, src_seq, n=None):
```

Comment:
I think `n` should be 1 by default because at line 67 you will do `int(n)`.

Reply:
For sure. I checked that value on the application I'm building using this library, but it would be better to have in the method definition to ensure a safer running.